src/ray/raylet/worker_pool_test.cc

```diff
@@ -1211,7 +1211,7 @@ TEST_F(WorkerPoolDriverRegisteredTest, MaxSpillRestoreWorkersIntegrationTest) {
       started_restore_processes.push_back(last_restore_process);
     }
     // Register workers with 10% probability at each time.
-    if (rand() % 100 < 10) {
+    if (rand() % 100 < 10) {  // NOLINT
```

Comment:
just a nit, not a merge blocker, next time it might be better to put the explanation beside
eg:
```
if (rand() % 100 < 10) { // NOLINT(balabala)
```